src/intel_driver.h

```diff
@@ -241,4 +242,6 @@ struct intel_region {
 
 #define IS_GEN10(device_info)           (device_info->gen == 10)
 
+#define IS_CNL(device_info)             (device_info->is_cannonlake)
```

Comment:
IS_GEN10() is used in the previous commits for CNL, how about to remove IS_CNL() and use IS_GEN10() instead?
we can remove is_cannonlake flag from struct intel_device_info if do so.